parallel-install/example/example.go

```diff
@@ -150,6 +145,22 @@ func main() {
 	if err != nil {
 		log.Fatalf("Failed to uninstall Kyma: %v", err)
 	}
+
+	postUninstallerCfg := postuninstaller.Config{
+		Log:              installationCfg.Log,
+		KubeconfigSource: installationCfg.KubeconfigSource,
+	}
+
+	postUninstaller, err := postuninstaller.NewPostUninstaller(postUninstallerCfg, commonRetryOpts)
+	if err != nil {
+		log.Fatalf("Failed to create Kyma post-uninstaller: %v", err)
+	}
+
+	uninstallResult, err := postUninstaller.UninstallCRDs()
+	if err != nil || len(uninstallResult.NotDeleted) > 0 {
```

Comment:
is there a possibility that err is nil but still there are some not deleted CDRs or other way around?

Reply:
It was not designed as fail-fast approach, so we would always end up with some data in `Output`. If `err` was nil all CRDs should land in either `Installed` or `NotInstalled` state containers.